docs/citation.rst

```diff
@@ -0,0 +1,26 @@
+.. _citation:
+
+**************
+How to cite us
+**************
+
+Please cite this article when using FAST-OAD in your research works:
+
+C. David, S. Delbecq, S. Defoort, P. Schmollgruber, E. Benard and V. Pommier-Budinger:
+*"From FAST to FAST-OAD: An open source framework for rapid Overall Aircraft Design"*,
+IOP Conference Series: Materials Science and Engineering, vol. 1024, n. 1, DOI: 10.1088/1757-899x/1024/1/012062
+
+.. code:: bib
+
+    @article{David_2021,
```

Comment:
Il n'y a généralement pas de underscore dans l'identifiant.

Reply:
Ok thank you,  it has been changed and a new commit was issued